packaging/install/cluster-operator/048-Crd-kafkamirrormaker2.yaml

```diff
@@ -262,6 +262,9 @@ spec:
                             enabled:
                               type: boolean
                               description: Whether automatic restart for failed connectors and tasks should be enabled or disabled.
+                            maxRestarts:
+                              type: integer
```

Comment:
What if I want to be explicit and set maxRetries to infinite in my CR? I think we should allow null at the schema level or use -1, which I think would be better to represent infinite retries.

Reply:
Why would you want it? It is infinite by default and there is no need to set it. (Also, this is how the proposal was approved)

Reply:
Sometimes, it is better to explicitly set a configuration to avoid issue in case the default value changes. Then, it is weird that there is a valid value that I cannot set through the CR.

Reply:
The use case is in my previous comment. That said, if this has been voted, then it's fine.